### PR TITLE
Honkenstein (DNM)

### PR DESCRIPTION
**Pull request by MMMiracles**
I mean it involves a castle sorta I swear I'll actually finish this and not just treat this PR as a tease.

The crew's gateway gets a set of coordinates leading them to what appears to be the remains of a russian fleet, only to discover its actually where the nazis have been hiding for all these years, on the moon.

It's more or less going to be wolfenstein but in space. I'm currently in the process of creating a bunch of assets before actually putting a map together. At worst if I loose steam on the project there will be some new, if slightly questionable, decorations for map makers.

All german clothing is ported from /vg/station, credit to @Heredth as hes the one who made the initial pull request for them.

I'll be uploading finished assets as I go to an imgur album here for easier viewing.
http://imgur.com/a/ScN4M

### CODE REVIEW

**Comment by Zilenan91** [comment]
>Space Nazis

Yes

File: code/modules/awaymissions/mission_code/raisearm.dm [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+/obj/structure/pillar
+	name = "pillar"
+	desc = "A reaaaaaally tall pillar"
+	icon = 'icons/obj/pillar.dmi'
+	icon_state = "pillarbottom"
+	anchored = 1
+	density = 1
+	layer = 10
+
```
Comment (KazeEspada) [inline code/modules/awaymissions/mission_code/raisearm.dm]: Needs to be destructable.

**Comment by tedward1337** [comment]
I thought this was going to be frankenhonk 
Was a good surprise to see space nazi's

**Comment by KazeEspada** [comment]
https://www.youtube.com/watch?v=5IojMhNoauk

**Comment by Dorsisdwarf** [comment]
Space Nazi mobs, hell yes.

**Comment by Impisi** [comment]
Oh, are you finally adding in the HoS cape?

**Comment by Supermichael777** [comment]
banana swastika

**Comment by RemieRichards** [comment]
banswasnanika

**Comment by MMMiracles** [comment]
I've started actually mapping now, I'll deal with conflicts after everything is finished.

File: code/modules/awaymissions/mission_code/raisearm.dm (hunk 1 above)
Comment (KazeEspada) [inline code/modules/awaymissions/mission_code/raisearm.dm]: If I can't knock over a pillar onto mecha hitler. Then I am leaving.

**Comment by Supermichael777** [comment]
needs communist lizard freedom fighter mimes(la résistance muet)

**Comment by Mark9013100** [comment]
ZHERE ARE NEIN NOTSEES ON ZE MOON

**Comment by MMMiracles** [comment]
In terms of actual map progress, I'd say I'm about 40% done. Will be adding the newer assets to the imgur album soon.

**Comment by MMMiracles** [comment]
Its almost done, probably gonna look into getting a test merge for a few rounds to get some balance feedback before I do the finishing touches.

File: code/game/objects/structures/lattice.dm [hunk 2]
```diff
@@ -1,85 +1,85 @@
-/obj/structure/lattice
```
Comment (MMMiracles) [inline code/game/objects/structures/lattice.dm]: i dont really even know what

**Comment by Cheridan** [comment]
stop trying to add swastikas to ss13

**Comment by MMMiracles** [comment]
space nazis are canon you can't stop me

File: code/modules/awaymissions/mission_code/raisearm.dm [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+/obj/structure/flag/cyka
+	name = "dusty flag"
+	desc = "A striped relic of a time long before. Smells faintly of vodka."
+	icon_state = "cyka"
+
+/obj/structure/flag/attack_hand(mob/user)
+	if(ripped)
+		return
+	var/temp_loc = user.loc
+	if((user.loc != temp_loc) || ripped )
```
Comment (KazeEspada) [inline code/modules/awaymissions/mission_code/raisearm.dm]: This looks like a shitty do after

File: code/modules/awaymissions/mission_code/raisearm.dm [hunk 4]
```diff
@@ -0,0 +1,443 @@
...
+	helmet = /obj/item/clothing/head/helmet/stalhelm
+	has_id = 0
+
+
+//space billboards
+
+/obj/structure/billboard
+	name = "Space billboard"
+	desc = "The pinnancle of space-advertisement."
+	icon = 'icons/obj//banters/billboard.dmi'
```
Comment (Dorsisdwarf) [inline code/modules/awaymissions/mission_code/raisearm.dm]: ?

File: code/modules/awaymissions/mission_code/raisearm.dm [hunk 5]
```diff
@@ -0,0 +1,443 @@
...
+
+/obj/structure/banner/cyka
+	name = "russian banner"
+	desc = "A banner. Smells faintly of vodka."
+	icon_state = "cyka"
+
+/obj/structure/banner/assblastusa
+	name = "patriotic banner"
+	desc = "A banner depicting a bald eagle."
+	icon_state = "assblastusa"
```
Comment (Dorsisdwarf) [inline code/modules/awaymissions/mission_code/raisearm.dm]: *american* banner

Also where's our banner you can't leave the world's uppitiest aircraft carrier out of this

File: code/modules/awaymissions/mission_code/raisearm.dm (hunk 4 above)
Comment (Cheridan) [inline code/modules/awaymissions/mission_code/raisearm.dm]: he couldn't handle the banter

**Comment by KorPhaeron** [comment]
@MMMiracles 

Are you gonna finish this?

**Comment by MMMiracles** [comment]
I will eventually, been taking breaks between sessions so I don't burn out on it.

Early-mid April is kind of my goal at this point.

**Comment by KorPhaeron** [comment]
This was killed by the turf rewrite probably. I'll re-open it if I'm wrong.

**Comment by PKPenguin321** [comment]
*bagpipes*
;-;7
You were a brave PR. You were just too beautiful for this world.
Farewell.

**Comment by duncathan** [comment]
I'm sorry
I'm so, so sorry

**Comment by MMMiracles** [comment]
Wir werden aus der Asche steigen.

**Comment by tedward1337** [comment]
Fucking kill, rip

**Pull request closed without merging**